[2025-02-13T06:22:48.638+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_dataset.transform_dataset manual__2025-02-13T06:22:44.283127+00:00 [queued]>
[2025-02-13T06:22:48.647+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_dataset.transform_dataset manual__2025-02-13T06:22:44.283127+00:00 [queued]>
[2025-02-13T06:22:48.648+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-13T06:22:48.659+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonVirtualenvOperator): transform_dataset> on 2025-02-13 06:22:44.283127+00:00
[2025-02-13T06:22:48.667+0000] {standard_task_runner.py:57} INFO - Started process 914 to run task
[2025-02-13T06:22:48.670+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'download_dataset', 'transform_dataset', 'manual__2025-02-13T06:22:44.283127+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/download_dataset_dag.py', '--cfg-path', '/tmp/tmp0ducp06m']
[2025-02-13T06:22:48.675+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask transform_dataset
[2025-02-13T06:22:48.724+0000] {task_command.py:416} INFO - Running <TaskInstance: download_dataset.transform_dataset manual__2025-02-13T06:22:44.283127+00:00 [running]> on host 081769f0e6bd
[2025-02-13T06:22:49.037+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='download_dataset' AIRFLOW_CTX_TASK_ID='transform_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T06:22:44.283127+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T06:22:44.283127+00:00'
[2025-02-13T06:22:49.038+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvf1ya_2xi
[2025-02-13T06:22:49.050+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T06:22:50.157+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 270ms
[2025-02-13T06:22:50.158+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvf1ya_2xi, clear=False, no_vcs_ignore=False, global=False)
[2025-02-13T06:22:50.158+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-02-13T06:22:50.158+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.2.1, setuptools==68.2.0, wheel==0.41.2
[2025-02-13T06:22:50.159+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-02-13T06:22:50.205+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvf1ya_2xi/bin/pip install -r /tmp/venvf1ya_2xi/requirements.txt
[2025-02-13T06:22:50.216+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T06:22:52.133+0000] {process_utils.py:190} INFO - Collecting pandas (from -r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.133+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pandas from https://files.pythonhosted.org/packages/f8/7f/5b047effafbdd34e52c9e2d7e44f729a0655efafb22198c45cf692cdc157/pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2025-02-13T06:22:52.136+0000] {process_utils.py:190} INFO -   Using cached pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)
[2025-02-13T06:22:52.252+0000] {process_utils.py:190} INFO - Collecting python-dateutil>=2.8.2 (from pandas->-r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.253+0000] {process_utils.py:190} INFO -   Obtaining dependency information for python-dateutil>=2.8.2 from https://files.pythonhosted.org/packages/ec/57/56b9bcc3c9c6a792fcbaf139543cee77261f3651ca9da0c93f5c1221264b/python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata
[2025-02-13T06:22:52.256+0000] {process_utils.py:190} INFO -   Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
[2025-02-13T06:22:52.338+0000] {process_utils.py:190} INFO - Collecting pytz>=2020.1 (from pandas->-r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.338+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pytz>=2020.1 from https://files.pythonhosted.org/packages/eb/38/ac33370d784287baa1c3d538978b5e2ea064d4c1b93ffbd12826c190dd10/pytz-2025.1-py2.py3-none-any.whl.metadata
[2025-02-13T06:22:52.340+0000] {process_utils.py:190} INFO -   Using cached pytz-2025.1-py2.py3-none-any.whl.metadata (22 kB)
[2025-02-13T06:22:52.374+0000] {process_utils.py:190} INFO - Collecting tzdata>=2022.1 (from pandas->-r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.375+0000] {process_utils.py:190} INFO -   Obtaining dependency information for tzdata>=2022.1 from https://files.pythonhosted.org/packages/0f/dd/84f10e23edd882c6f968c21c2434fe67bd4a528967067515feca9e611e5e/tzdata-2025.1-py2.py3-none-any.whl.metadata
[2025-02-13T06:22:52.377+0000] {process_utils.py:190} INFO -   Using cached tzdata-2025.1-py2.py3-none-any.whl.metadata (1.4 kB)
[2025-02-13T06:22:52.877+0000] {process_utils.py:190} INFO - Collecting numpy>=1.20.3 (from pandas->-r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.879+0000] {process_utils.py:190} INFO -   Obtaining dependency information for numpy>=1.20.3 from https://files.pythonhosted.org/packages/98/5d/5738903efe0ecb73e51eb44feafba32bdba2081263d40c5043568ff60faf/numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2025-02-13T06:22:52.880+0000] {process_utils.py:190} INFO -   Using cached numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.6 kB)
[2025-02-13T06:22:52.913+0000] {process_utils.py:190} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas->-r /tmp/venvf1ya_2xi/requirements.txt (line 1))
[2025-02-13T06:22:52.914+0000] {process_utils.py:190} INFO -   Obtaining dependency information for six>=1.5 from https://files.pythonhosted.org/packages/b7/ce/149a00dd41f10bc29e5921b496af8b574d8413afcd5e30dfa0ed46c2cc5e/six-1.17.0-py2.py3-none-any.whl.metadata
[2025-02-13T06:22:52.916+0000] {process_utils.py:190} INFO -   Using cached six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
[2025-02-13T06:22:52.968+0000] {process_utils.py:190} INFO - Using cached pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.4 MB)
[2025-02-13T06:22:53.045+0000] {process_utils.py:190} INFO - Using cached numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
[2025-02-13T06:22:53.073+0000] {process_utils.py:190} INFO - Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[2025-02-13T06:22:53.076+0000] {process_utils.py:190} INFO - Using cached pytz-2025.1-py2.py3-none-any.whl (507 kB)
[2025-02-13T06:22:53.079+0000] {process_utils.py:190} INFO - Using cached tzdata-2025.1-py2.py3-none-any.whl (346 kB)
[2025-02-13T06:22:53.082+0000] {process_utils.py:190} INFO - Using cached six-1.17.0-py2.py3-none-any.whl (11 kB)
[2025-02-13T06:22:53.406+0000] {process_utils.py:190} INFO - Installing collected packages: pytz, tzdata, six, numpy, python-dateutil, pandas
[2025-02-13T06:23:00.179+0000] {process_utils.py:190} INFO - Successfully installed numpy-1.24.4 pandas-2.0.3 python-dateutil-2.9.0.post0 pytz-2025.1 six-1.17.0 tzdata-2025.1
[2025-02-13T06:23:00.281+0000] {process_utils.py:190} INFO - 
[2025-02-13T06:23:00.282+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.2.1 -> 25.0.1
[2025-02-13T06:23:00.282+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-02-13T06:23:00.497+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvf1ya_2xi/bin/python /tmp/venvf1ya_2xi/script.py /tmp/venvf1ya_2xi/script.in /tmp/venvf1ya_2xi/script.out /tmp/venvf1ya_2xi/string_args.txt /tmp/venvf1ya_2xi/termination.log
[2025-02-13T06:23:00.510+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T06:23:01.625+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-13T06:23:01.639+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=download_dataset, task_id=transform_dataset, execution_date=20250213T062244, start_date=20250213T062248, end_date=20250213T062301
[2025-02-13T06:23:01.694+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-13T06:23:01.711+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
